--- a/flake.nix
+++ b/flake.nix
@@ -125,7 +125,7 @@
     };
   };
   outputs =
-    prevInputs@{
+    inputs@{
       fenix,
       flake-parts,
       home-manager,
@@ -134,11 +134,6 @@
       systems,
       ...
     }:
-    let
-      inputs = prevInputs // {
-        private-pkgs = builtins.getFlake "github:ms0503/private-pkgs.nix/90b9b2d3897e66ff8b008c2ba66262ddc1ca4720";
-      };
-    in
     flake-parts.lib.mkFlake { inherit inputs; } (
       { flake-parts-lib, withSystem, ... }:
       let
--- a/machines/a15/home-manager.nix
+++ b/machines/a15/home-manager.nix
@@ -32,17 +32,6 @@
       platformio
       stm32cubemx
     ];
-    pointerCursor = {
-      gtk.enable = true;
-      name = "Milk1";
-      package = pkgs.milk-cursor-themes;
-      size = 32;
-      x11.enable = true;
-    };
-    sessionVariables = {
-      HYPRCURSOR_SIZE = "32";
-      HYPRCURSOR_THEME = "Milk1";
-    };
   };
   imports = [
     ../../home-manager/desktop/hyprland
--- a/machines/default.nix
+++ b/machines/default.nix
@@ -4,7 +4,6 @@
   inherit (inputs)
     fenix
     nix-ros-overlay
-    private-pkgs
     self
     ;
   inherit (self.lib) mkHomeManagerConfiguration mkNixosSystem;
@@ -21,7 +20,6 @@
         overlays = [
           fenix.overlays.default
           nix-ros-overlay.overlays.default
-          private-pkgs.overlays.default
         ];
         system = "x86_64-linux";
         theme = "chalk";
@@ -35,7 +33,6 @@
         ];
         overlays = [
           fenix.overlays.default
-          private-pkgs.overlays.default
         ];
         system = "x86_64-linux";
         theme = "chalk";
--- a/machines/mainpc/home-manager.nix
+++ b/machines/mainpc/home-manager.nix
@@ -2,17 +2,6 @@
 {
   home = {
     packages = [ ];
-    pointerCursor = {
-      gtk.enable = true;
-      name = "Milk1";
-      package = pkgs.milk-cursor-themes;
-      size = 32;
-      x11.enable = true;
-    };
-    sessionVariables = {
-      HYPRCURSOR_SIZE = "32";
-      HYPRCURSOR_THEME = "Milk1";
-    };
  };
  imports = [
    ../../home-manager/desktop/hyprland
