--- a/flake.nix
+++ b/flake.nix
@@ -120,7 +120,7 @@
     };
   };
   outputs =
-    prevInputs@{
+    inputs@{
       fenix,
       flake-parts,
       home-manager,
@@ -129,11 +129,6 @@
       systems,
       ...
     }:
-    let
-      inputs = prevInputs // {
-        private-pkgs = builtins.getFlake "github:ms0503/private-pkgs.nix/71c325cefdfff07fb3a8648884c042aef2787bf7";
-      };
-    in
     flake-parts.lib.mkFlake { inherit inputs; } (
       { flake-parts-lib, withSystem, ... }:
       let
--- a/machines/a15/home-manager.nix
+++ b/machines/a15/home-manager.nix
@@ -22,17 +22,6 @@
       platformio
       stm32cubemx
     ];
-    pointerCursor = {
-      gtk.enable = true;
-      name = "Milk1";
-      package = pkgs.milk-cursor-themes;
-      size = 32;
-      x11.enable = true;
-    };
-    sessionVariables = {
-      HYPRCURSOR_SIZE = "32";
-      HYPRCURSOR_THEME = "Milk1";
-    };
   };
   imports = [
     ../../home-manager/desktop/hyprland
--- a/machines/default.nix
+++ b/machines/default.nix
@@ -3,7 +3,6 @@
 let
   inherit (inputs)
     fenix
-    private-pkgs
     self
     ;
   inherit (self.lib) mkHomeManagerConfiguration mkNixosSystem;
@@ -19,7 +18,6 @@
         ];
         overlays = [
           fenix.overlays.default
-          private-pkgs.overlays.default
         ];
         system = "x86_64-linux";
         theme = "chalk";
@@ -33,7 +31,6 @@
         ];
         overlays = [
           fenix.overlays.default
-          private-pkgs.overlays.default
         ];
         system = "x86_64-linux";
         theme = "chalk";
--- a/machines/mainpc/home-manager.nix
+++ b/machines/mainpc/home-manager.nix
@@ -2,17 +2,6 @@
 {
   home = {
     packages = [ ];
-    pointerCursor = {
-      gtk.enable = true;
-      name = "Milk1";
-      package = pkgs.milk-cursor-themes;
-      size = 32;
-      x11.enable = true;
-    };
-    sessionVariables = {
-      HYPRCURSOR_SIZE = "32";
-      HYPRCURSOR_THEME = "Milk1";
-    };
  };
  imports = [
    ../../home-manager/desktop/hyprland
