--- a/flake.nix
+++ b/flake.nix
@@ -168,20 +168,7 @@
     ];
   };
   outputs =
-    prevInputs@{
-      flake-parts,
-      nixpkgs,
-      systems,
-      ...
-    }:
-    let
-      inputs = nixpkgs.lib.mergeAttrsList [
-        prevInputs
-        {
-          private-pkgs = builtins.getFlake "github:ms0503/private-pkgs.nix/b11c013de609eb315bcff990d845e188cccb589e";
-        }
-      ];
-    in
+    inputs@{ flake-parts, systems, ... }:
     flake-parts.lib.mkFlake { inherit inputs; } (
       { withSystem, ... }:
       {
--- a/machines/default.nix
+++ b/machines/default.nix
@@ -2,10 +2,10 @@
 let
   inherit (inputs)
     fenix
-    private-pkgs
     self
     ;
   inherit (self.lib) mkHomeManagerConfiguration mkNixosSystem;
+  private-pkgs.overlays.default = _: _: { };
 in
 {
   flake = {
